core/src/config/stump_config.rs

```diff
@@ -65,219 +67,91 @@ use defaults::*;
 ///   let core = StumpCore::new(config).await;
 /// }
 /// ```
-#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[derive(StumpConfigGenerator, Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[config_file_location(self.get_config_dir().join("Stump.toml"))]
 pub struct StumpConfig {
 	/// The "release" | "debug" profile with which the application is running.
+	#[default_value("debug".to_string())]
+	#[env_key(PROFILE_KEY)]
+	#[validator(do_validate_profile)]
 	pub profile: String,
+
 	/// The port from which to serve the application (default: 10801).
+	#[default_value(10801)]
+	#[env_key(PORT_KEY)]
 	pub port: u16,
+
 	/// The verbosity with which to log errors (default: 0).
+	#[default_value(0)]
+	#[env_key(VERBOSITY_KEY)]
 	pub verbosity: u64,
+
 	/// Whether or not to pretty print logs.
+	#[default_value(true)]
+	#[env_key(PRETTY_LOGS_KEY)]
 	pub pretty_logs: bool,
+
 	/// An optional custom path for the database.
+	#[default_value(None)]
+	#[env_key(DB_PATH_KEY)]
 	pub db_path: Option<String>,
+
 	/// The client directory.
+	#[default_value("./dist".to_string())]
```

Comment:
Similarly as above,this default value should be `./client` as the second part of fixing the bug